tools/pr_comments.py

```diff
@@ -0,0 +1,35 @@
+# This file is part of the EESSI build-and-deploy bot,
+# see https://github.com/EESSI/eessi-bot-software-layer
+#
+# The bot helps with requests to add software installations to the
+# EESSI software layer, see https://github.com/EESSI/software-layer
+#
+# author: Kenneth Hoste (@boegel)
+# author: Bob Droege (@bedroge)
+# author: Hafsa Naeem (@hafsa-naeem)
+# author: Thomas Roeblitz (@trz42)
+#
+# license: GPLv2
+#
+import re
+
+
+def get_comment(pr, job_id):
+    comments = pr.get_issue_comments()
+    for comment in comments:
+        # NOTE adjust search string if format changed by event
+        #        handler (separate process running
+        #        eessi_bot_event_handler.py)
+        cms = f".*submitted.*job id `{job_id}`.*"
+        # cms = f".*{search_pattern}.*"
```

Comment:
I'd suggest to use a generic version, for example, the one that is commented out in line 24 now. If we need a more specific one, we could always define an additional function, say `get_submitted_job_comment()` which could be defined as follows

```python
def get_submitted_job_comment(pr, job_id):
    search_pattern = f"submitted.*job_id `{job_id}`"
    return get_comment(pr, search_pattern)
```